pentagon/migration/migrations/migration_1_2_0.py

```diff
@@ -0,0 +1,163 @@
+
+import pentagon
+from pentagon import migration
+from collections import OrderedDict
+from pentagon.migration import *
+
+
+class Migration(migration.Migration):
+    _starting_version = '1.2.0'
+    _ending_version = '2.0.0'
+
+    def run(self):
+
+        # Create inventory directory if it exists
+        inventory_dir = '{}/inventory'.format(self._infrastructure_repository)
+        if not os.path.isdir(inventory_dir):
+            os.mkdir(inventory_dir)
+
+        # Move default
+        if os.path.exists('{}/default'.format(self._infrastructure_repository)):
+            self.move('default', "inventory/default")
+
+        self.delete('config/local/env-vars.sh')
+        if os.path.exists('{}/config'.format(self._infrastructure_repository)):
+            self.move('config/', 'inventory/default/config')
+
+        for item in self.inventory:
+
+            inventory_path = "inventory/{}".format(item)
+            logging.debug('Inventory Path: {}'.format(inventory_path))
+
+            if os.path.exists('{}/account/vars.yml'.format(self._infrastructure_repository)):
+                account_vars = self.YamlEditor('{}/account/vars.yml'.format(inventory_path)).get()
+            else:
+                account_vars = OrderedDict()
+
+            if os.path.exists('{}/account/vars.sh'.format(self._infrastructure_repository)):
+                account_vars_sh = self.get_file_content('{}/account/vars.sh'.format(inventory_path)).get()
+                account_vars = OrderedDict()
+
+                for line in account_vars_sh.split('\n'):
+                    line = line.replace('export ', '')
+                    llist = line.split('=', 1)
+                    account_vars[llist[0]] = llist[1]
+            else:
+                account_vars = OrderedDict()
+
+            config_vars_yml = self.YamlEditor('{}/config/local/vars.yml'.format(inventory_path))
+            config_vars_yml.update(account_vars)
+            config_vars_yml.update(account_vars)
```

Comment:
I think you could get rid of one of these lines.

Reply:
Actually, you caught a bug.